content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -119,7 +119,7 @@ This command also specifies the output file. In this case, the output file is `c
 
 In Corda 4, this process was much more involved. Now you can install the application on the network with a single command:
 
-  `corda-cli package install -n solar-system cordaSolarSystem.cpb`
+  `corda-cli package install -n solar-system corda5-hello-solarsystem.cpb`
```

Comment:
I'm pretty sure the cpb will be found in one of the build folders. We'd need to check the gradle config to see where the cpb is output and what it's called.

Reply:
Yes, indeed you get the workflow and the contract packages under the corresponding build/lib folders, however both the package and the workflow packages (`.cpk`) are contained in a bundle (`.cpb`) as a result of Step 5, which demonstrates the `cordapp-builder` CLI utility.